site/content/en/docs/installation/_index.md

```diff
@@ -64,6 +64,15 @@ kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases
 To allow [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator)
 to scrape metrics from kueue components, run the following command:
 
+> **Note**: This feature depends on [servicemonitor CRD](https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/setup/0servicemonitorCustomResourceDefinition.yaml), please ensure that CRD is installed first.
+> We can follow `https://prometheus-operator.dev/docs/prologue/quick-start/` to install it.
+> 
+  ```sh
+  git clone https://github.com/prometheus-operator/kube-prometheus.git
+  kubectl apply -f github.com/prometheus-operator/kube-prometheus/manifests/setup/0servicemonitorCustomResourceDefinition.yaml
+  ```
+
```

Comment:
```suggestion
```

Is this still needed? Even if we perform this command, the Prometheus feature doesn't work well, right?
For sure, once we perform this command, the Kueue with Prometheus installation would succeed.

Reply:
> For sure, once we perform this command, the Kueue with Prometheus installation would succeed.

Yes, with this command, the installation of kueue will be succeed.

Reply:
Even if performing this command, the kueue with Prometheus wouldn't work fine.
So, I don't think that this command would be worth it.